Sources/Authentication/AppAuthSession.swift

```diff
@@ -70,16 +72,27 @@ public final class AppAuthSession: LoginSession {
         }
     }
     
-    
+    /// Finishes the login dialog
+    ///
+    /// - Parameters:
+    ///     - callback: the URL to an authorization endpoint you started the flow with.
+    ///     - endpoint: the token endpoint to query to receive tokens back as the final action in the flow.
     @MainActor
-    public func finalise(callback url: URL) async throws -> TokenResponse {
+    public func finalise(callback url: URL, endpoint: URL) async throws -> TokenResponse {
```

Comment:
Do we need to pass `endpoint` in here? 

We're changing the interface for consumers of the packages, so I want to make sure it's necessary.

I think we can retrieve this value from the `LoginSessionConfiguration` which the consumer has already passed in.

Reply:
Nope, wood from the trees moment 😅 - we can get that from the `LoginSessionConfiguration` object